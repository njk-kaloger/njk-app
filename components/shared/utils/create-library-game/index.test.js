import createLibraryGame from "./";

it("create a library game", () => {
  expect(
    createLibraryGame(
      "The Witcher 3: Wild Hunt",
      "PC",
      "RPG",
      "https://images-na.ssl-images-amazon.com/images/I/71NrauAMwbL._SY445_.jpg",
      "https://thumbs.gfycat.com/ExcitableLimitedBufeo-size_restricted.gif",
      "The WitcherÂ® 3: Wild Hunt is a story-driven, next-generation open world role-playing game, set in a visually stunning fantasy universe, full of meaningful choices and impactful consequences. You play as Geralt of Rivia, a monster hunter tasked with finding a child from an ancient prophecy."
    )
  ).toMatchSnapshot();
});
