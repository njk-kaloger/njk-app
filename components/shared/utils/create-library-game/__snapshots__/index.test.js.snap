// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a library game 1`] = `
Object {
  "animatedImage": "The WitcherÂ® 3: Wild Hunt is a story-driven, next-generation open world role-playing game, set in a visually stunning fantasy universe, full of meaningful choices and impactful consequences. You play as Geralt of Rivia, a monster hunter tasked with finding a child from an ancient prophecy.",
  "completionRate": 0,
  "desc": "RPG",
  "genre": "https://images-na.ssl-images-amazon.com/images/I/71NrauAMwbL._SY445_.jpg",
  "id": "PC-The Witcher 3: Wild Hunt",
  "image": "https://thumbs.gfycat.com/ExcitableLimitedBufeo-size_restricted.gif",
  "platform": "PC",
  "title": "The Witcher 3: Wild Hunt",
}
`;
